modules/core/24-host/keys.go

```diff
@@ -218,7 +218,7 @@ func PacketReceiptKey(portID, channelID string, sequence uint64) []byte {
 	return []byte(PacketReceiptPath(portID, channelID, sequence))
 }
 
-func channelPath(portID, channelID string) string {
+func parseChannelPath(portID, channelID string) string {
```

Comment:
I'd revert this one, we're not parsing really, we're constructing the path.

Reply:
done